CONTRIBUTING.md

```diff
@@ -148,10 +148,10 @@ In this section we are going to start a new workspace to work on workspace loade
 The workspace could be created and started from a devfile using `chectl`:
 
 ```bash
-$ chectl workspace:start --devfile=https://raw.githubusercontent.com/eclipse/che/master/workspace-loader/devfile.yaml
+$ chectl workspace:start --devfile=https://raw.githubusercontent.com/eclipse/che-workspace-loader/master/devfile.yaml
 ```
 
-or as a factory `https://<CheInstance>/f?url=https://raw.githubusercontent.com/eclipse/che/master/workspace-loader/devfile.yaml`
+or as a factory `https://<CheInstance>/f?url=https://raw.githubusercontent.com/eclipse/che-workspace-loader/master/devfile.yaml
 
 At workspace start, Che will clone Che source code (including the workspace-loader)
```

Comment:
No longer true. And the devfile is broken now as it has Che project with `workspace-loader` as `sparseCheckoutDir`, which does not exist now.